pkg/services/alerting/alerting_usage_test.go

```diff
@@ -19,7 +20,7 @@ func TestAlertingUsageStats(t *testing.T) {
 
 	ae.Bus.AddHandler(func(query *models.GetAllAlertsQuery) error {
 		var createFake = func(file string) *simplejson.Json {
-			content, err := ioutil.ReadFile(file)
+			content, err := ioutil.ReadFile(filepath.Clean(file))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.